### ISSUE TITLE
Feature: Add Imperial Assault dice

### ISSUE THREAD

**Issue by Kregoras** [BernhardPosselt/foundryvtt-special-dice-roller#59]
Hi!

Love your module!
I've been looking to have something like this for Imperial Assault and, having already descent dice, it should be quite easy to add those. Could you please integrate those dice on your module? Thank you very much.

**Comment by BernhardPosselt** [comment]
No, I neither have the license for the dice symbols, nor images, nor the rules. If you want them, please contribute them to the repo, there's documentation here https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/blob/master/docs/Roller.md
